test/test_integration.rb

```diff
@@ -102,9 +102,10 @@ def test_authorizer_silent
   end
 
   def test_prepare_invalid_syntax
-    assert_raise(SQLite3::SQLException) do
+    exception = assert_raise(SQLite3::SQLException) do
       @db.prepare "select from foo"
     end
+    assert_equal("near \"from\": syntax error\n  select from foo\n         ^--- error here", exception.message)
```

Comment:
Heredocs are usually best for that sort of things.

```suggestion
    assert_equal(<<~MSG, exception.message)
      near "from": syntax error
        select from foo
                 ^--- error here
      MSG
```